docs/reference/esql/processing-commands/change_point.asciidoc

```diff
@@ -0,0 +1,47 @@
+[discrete]
+[[esql-change_point]]
+=== `CHANGE_POINT`
+
+[NOTE]
+====
+The `CHANGE_POINT` command requires a [platinum license](https://www.elastic.co/subscriptions).
+
```

Comment:
You have failed to close the note block, so most of the contents below are included in the note.